pkg/roachprod/vm/aws/aws.go

```diff
@@ -757,6 +753,13 @@ func (p *Provider) Create(
 	return vmList, nil
 }
 
+func DefaultZones(geoDistributed bool) []string {
+	if geoDistributed {
+		return defaultZones
+	}
+	return defaultZones[:1]
```

Comment:
Nit: that's an overly complicated way of saying `defaultZones[0]`; preserves the symmetry with the other provider(s) :)

Reply:
Yeah, it bothered me while I was refactoring the code, and I forgot to come back to it.

Switched in Azure and GCE to the more approachable `[]string{defaultZones[0]}` that was already used in Azure and that I went for IBM.